Project-Descriptions-and-Plans/CVA6/CVA6-main-plan-approved-2024+.md

```diff
@@ -0,0 +1,524 @@
+- The University of Minho
+- Zero-Day Labs
+- PULP team (ETH Zürich and U. Bologna)
+- 10xEngineers
+- MU-Electronics
+- CEA
+- PlanV
+- Axelera
+- A partner of the TRISTAN project
+
+## Project Structure
+
+The CVA6 project has evolved to a main project and several sub-projects as more CVA6 configurations are envisioned to target TRL5.
+
+### Common goals
+
+Despite the number of sub-projects, the following goals will be pursued by the main and sub-projects:
+- Optimize the effort sharing and reuse between configurations
+- Keep a common RTL database (with parameters)
+- Manage a common documentation
+- Still be able to accept relevant evolutions (but be more selective than in the past)
+
+The scopes of the main, sub-, and sister projects are described below:
+
+### Main project
+
+The main CVA6 project includes:
+- The synchronization with the sub-projects
+- The review and selection of proposed CVA6 evolutions<sup>2</sup>
+- The review of pull requests (PRs) from sister projects and the external world
+- The 2<sup>nd</sup> level review of PRs from sub-projects, mostly to check that the PR integrates well in CVA6 repository
+- The maintenance of the GitHub-triggered CI flow, the CVA6 GitHub repository , the triage of external issues
+- Design activities that are not hosted in a sub- or sister project (MMU unification, FPGA optimizations...)
+- The definition of a release process, that is needed by sub-projects.
+
+Note:<br>
+<sup>2</sup> A sister project and/or a TWG gate can be recommended for major evolutions.
+
+### Sub-projects
+
+Each CVA6 configuration targetting the TRL5 maturity will launch a sub-project with a PA gate presented at a TWG meeting.
+
+The sub-projects mostly include verification activities.
+They also host design and documentation activities that are specific to the configuration.
+
+The PA gates of sub-projects include:
+- Quality targets (coverage targets, certification if any...)
+- Additions to the RTL freeze checklist as deemed necessary
+- Identification of reused/joint/new verification building blocks (DV plans, agents, sequences...)
+- Doc/Design/DVPlan/sequences creation/updates in synchronization with other configurations
+- 1<sup>st</sup> level review of PRs from the sub-project, mostly to check that the PR meets the sub-project goals and quality targets
+- Fix issues discovered in the sub-project
+- Release plan for the configuration, based on the release process
+
+The sub-project PA gates are prepared and presented by their respective leaders.
+Prior to presenting the gate to the TWG, it is recommended to review the sub-project and PA gate document in a CVA6 meeting.
+
+At the time of writing, the envisioned sub-projects are:
+- CV32A60AX<sup>3</sup>, a 32-bit application core, led by 10xEngineers
+- CV32A65X, a 32-bit embedded core with dual issue<sup>4</sup>, led by TSS
+- CV64A60AV, a 64-bit application core with an interface to a vector coprocessor, led by Axelera
```

Comment:
Might mention X-IF support in the ones that support it as that is I think a major feature

Reply:
I'll complete.